sequel/Gemfile

```diff
@@ -0,0 +1,29 @@
+source 'https://rubygems.org'
+
+module ::Kernel
+  def sequel_master?
+    ENV["SEQUEL_MASTER"] == '1'
+  end
+
+  def mysql2_prepared_statements?
+    ENV["MYSQL2_PREPARED_STATEMENTS"] == '1'
+  end
+end
+
+if sequel_master?
+  gem 'sequel', github: 'jeremyevans/sequel', branch: 'master'
+else
+  gem 'sequel', path: '/sequel'
+end
+
+gem 'rails', '~> 5.0.0.rc1'
```

Comment:
Adding only activerecord gives the error:
`block in replace_gem': railties is not part of the bundle. Add it to Gemfile. (Gem::LoadError)` 
So I added `railties` to the gemfile as well and then its working!